pennylane/transforms/decompositions/two_qubit_unitary.py

```diff
@@ -429,7 +442,10 @@ def _decomposition_3_cnots(U, wires):
     evs, _ = math.linalg.eig(gammaU)
 
     # We will sort the angles so that results are consistent across interfaces.
-    angles = math.sort([math.angle(ev) for ev in evs])
+    angles = [math.angle(ev) for ev in evs]
+
+    if not qml.math.is_abstract(U):
+        angles = math.sort(angles)
```

Comment:
Worth mentioning that sorting is not compatible with jit because we don't yet know the magnitude of each angle.